moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -273,6 +254,11 @@ void JogCalcs::haltOutgoingJogCmds()
   halt_outgoing_jog_cmds_ = true;
 }
 
+void JogCalcs::proceedOutgoingJogCmds()
+{
+  halt_outgoing_jog_cmds_ = true;
+}
```

Comment:
About this function name again -- it seems backward. `halt_outgoing_jog_cmds_ == true` means the jogger will halt. But the name of this function implies it will do the reverse

Reply:
Good catch! Of course this should be `false`